project/Dependencies.scala

```diff
@@ -12,7 +12,7 @@ object Dependencies extends CoreDependencies {
 
   val commonsIO = "commons-io" % "commons-io" % "2.4"
 
-  val datastore = "org.allenai" %% "datastore" % "2015.04.02-0"
+  val datastore = "org.allenai" %% "datastore" % "1.0.0"
```

Comment:
Can you add a TODO regarding the circular dependency from the indexing utils library? This needs to be broken.